packages/local-build-plugin/src/ios.ts

```diff
@@ -23,10 +23,18 @@ export async function buildIosAsync(
     logger,
     logBuffer,
     uploadArtifact: async ({ artifact, logger }) => {
-      if (artifact.type === ManagedArtifactType.APPLICATION_ARCHIVE) {
-        return await prepareArtifacts(artifact.paths, logger);
-      } else if (artifact.type === ManagedArtifactType.BUILD_ARTIFACTS) {
-        return await prepareArtifacts(artifact.paths, logger);
+      if (isManagedArtifact(artifact)) {
+        if (
+          artifact.type === ManagedArtifactType.XCODE_BUILD_LOGS &&
+          artifact.runStatus !== 'errored'
```

Comment:
Would it make sense to just always upload Xcode build logs? It would simplify code greatly (no tracking of `runStatus`, uploading artifact in `finally`)

Reply:
Maybe it would. Would it be ok for you as a user if they were always uploaded for you local builds (copied to project dir) 🤔